geometry/SettingsViewController.swift

```diff
@@ -31,25 +29,32 @@ class SettingsViewController: UIViewController
     @IBOutlet weak var labelGreen: UILabel!
```

Comment:
Similar variables are better to start from different names: greenLabel, blueLabel, etc. This way they are easy to find with autocomplete tools of IDE.